### ISSUE TITLE
✔️ 🐞 Bug Report: X Error on form file upload after updating to v3.11

### ISSUE THREAD

**Issue by up-creator** [elementor/elementor#21341]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Add file upload to form and the form shows "X - Error" when send.
Rollback to 3.10.3 (Elementor Pro) and the form works again.

### Steps to reproduce

Use Elementor Pro 3.11.0 and try to use form with file upload field.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: Percona Server (GPL), Release 43, Revision c1b94a6cfd7 v5.7.40-43
	PHP Version: 8.0.27
	PHP Memory Limit: 512M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://byghaandvaerk.dk
	Home URL: https://byghaandvaerk.dk
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: da-DK
	Timezone: Europe/Copenhagen
	Admin Email: wp@uptime.dk
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: da_DK
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.0.7
		Author: WP Engine

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Duplicate Page
		Version: 4.5.1
		Author: mndpsingh287

	Duplicator
		Version: 1.5.2.1
		Author: Snap Creek

	Dynamic.ooo - Dynamic Content for Elementor
		Version: 1.16.9
		Author: Dynamic.ooo

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Ele Custom Skin Pro
		Version: 3.1.0
		Author: Dudaster.com

	Elementor
		Version: 3.11.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.10.3
		Author: Elementor.com

	Favicon by RealFaviconGenerator
		Version: 1.3.27
		Author: Philippe Bernard

	JetEngine
		Version: 3.1.1
		Author: Crocoblock

	JetSmartFilters
		Version: 3.0.4
		Author: Crocoblock

	LiteSpeed Cache
		Version: 5.3.2
		Author: LiteSpeed Technologies

	Mouseflow for Wordpress
		Version: 5.1.2
		Author: Mouseflow

	Really Simple SSL
		Version: 6.2.0
		Author: Really Simple Plugins

	Wordfence Security
		Version: 7.8.2
		Author: Wordfence

	WP All Import - ACF Add-On
		Version: 3.3.8
		Author: Soflyy

	WP All Import Pro
		Version: 4.7.8
		Author: Soflyy

	WPCode Lite
		Version: 2.0.7
		Author: WPCode

	WP Mail SMTP
		Version: 3.7.0
		Author: WPForms

	Yoast SEO
		Version: 20.1
		Author: Team Yoast


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com


== Elements Usage ==
	
	archive : 3
		heading : 4
		jet-listing-grid : 4
		jet-smart-filters-pagination : 1
		text-editor : 4
	footer : 1
		button : 1
		icon-list : 1
		nav-menu : 2
		text-editor : 7
	header : 1
		icon-list : 1
		nav-menu : 1
		theme-site-logo : 1
	loop : 3
		button : 2
		heading : 1
		icon-list : 1
		image : 1
		post-info : 1
		spacer : 1
		text-editor : 4
		theme-post-featured-image : 1
		theme-post-title : 2
	page : 3
		button : 3
		heading : 7
		image : 3
		media-carousel : 1
		text-editor : 17
	popup : 3
		button : 1
		form : 2
		heading : 2
		html : 2
		image : 1
		text-editor : 1
	section : 7
		button : 5
		heading : 5
		icon : 8
		image : 1
		jet-listing-grid : 2
		media-carousel : 1
		posts : 2
		spacer : 1
		text-editor : 13
	widget : 0
		accordion : 1
		button : 1
		jet-listing-grid : 1
	wp-page : 16
		accordion : 3
		button : 62
		heading : 71
		html : 4
		icon : 14
		icon-box : 14
		icon-list : 8
		image : 40
		image-carousel : 6
		jet-listing-grid : 17
		menu-anchor : 2
		posts : 6
		shortcode : 1
		spacer : 12
		text-editor : 201
	single-post : 3
		button : 1
		divider : 2
		dyncontel-acf-google-maps : 1
		heading : 8
		icon : 2
		icon-list : 1
		image : 1
		image-carousel : 2
		jet-listing-dynamic-field : 17
		jet-listing-dynamic-image : 1
		post-navigation : 1
		text-editor : 7
		theme-post-content : 2
	single-page : 1
		button : 3
		dyncontel-acf-google-maps : 1
		heading : 3
		icon : 1
		icon-box : 1
		jet-listing-dynamic-field : 5
		jet-listing-dynamic-link : 2
		jet-listing-grid : 2
		text-editor : 6
	jet-listing-items : 0
		button : 1
		heading : 2
		image : 3
		jet-listing-dynamic-field : 9
		jet-listing-dynamic-image : 1
		jet-listing-dynamic-link : 1
		text-editor : 2
	error-404 : 1
		heading : 3


== Indstillinger ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1


== Funktioner ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrationer ==
	
	google_maps: Aktiv


== Elementor Experiments ==
	Optimeret DOM-output: Inaktiv som standard
	Forbedret ressourceindlæsning : Inaktiv som standard
	Forbedret indlæsning af CSS ressourcer: Inaktiv som standard
	Inline Font Icons: Inaktiv som standard
	Forbedringer af tilgængeligheden: Inaktiv som standard
	Additional Custom Breakpoints: Inaktiv som standard
	admin_menu_rearrangement: Inaktiv som standard
	Flexbox Container: Inaktiv som standard
	Upgrade Swiper Library: Inaktiv som standard
	Default to New Theme Builder: Aktiv som standard
	Hello sidehoved og sidefod: Inaktiv som standard
	Editor V2: Inaktiv som standard
	Landingsside: Aktiv som standard
	Nested Elements: Inaktiv som standard
	Lazy Load baggrundsbilleder: Inaktiv som standard
	Gem som standard: Aktiv som standard
	Page Transitions: Aktiv som standard
	Notes: Aktiv som standard
	Loop: Inaktiv som standard
	Modtagne formularer: Aktiv som standard
	Scroll Snap: Aktiv som standard


== Log ==
	
Log: showing 20 of 272023-02-13 09:14:25 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2023-02-13 09:14:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.4',
  'to' => '3.10.2',
)]
2023-02-13 09:18:02 [info] elementor-pro::elementor_pro_updater Started 
2023-02-13 09:18:02 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-13 09:18:02 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-13 09:18:02 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Start  
2023-02-13 09:18:02 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.1',
  'to' => '3.10.3',
)]
2023-02-13 09:18:04 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Finished 
2023-02-13 09:18:04 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2023-02-13 09:18:06 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2023-02-13 09:18:06 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.1',
  'to' => '3.10.3',
)]
2023-02-14 13:34:00 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-14 13:34:01 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-14 13:34:02 [info] elementor::elementor_updater Started 
2023-02-14 13:34:02 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-14 13:34:02 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-14 13:34:02 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-14 13:34:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-14 13:34:03 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-14 13:34:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]

PHP: showing 10 of 10PHP: 2023-02-13 09:14:25 [notice X 1][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 09:14:27 [error X 2][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php::48] Uncaught ReflectionException: Method ElementorPro\Modules\ThemeBuilder\Documents\Section::get_site_editor_type() does not exist in /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php:48
Stack trace:
#0 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(48): ReflectionClass->getMethod()
#1 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(129): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document::get_site_editor_type_bc()
#2 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/categories/create.php(94): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document::get_create_url()
#3 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/categories/create.php(67): Elementor\Core\Common\Modules\Finder\Categories\Create->create_item_url_by_document_class()
#4 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/base-category.php(92): Elementor\Core\Common\Modules\Finder\Categories\Create->get_category_items()
#5 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Core\Common\Modules\Finder\Base_Category->get_init_settings()
#6 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Base_Object->ensure_settings()
#7 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/module.php(107): Elementor\Core\Base\Base_Object->get_settings()
#8 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Core\Common\Modules\Finder\Module->get_init_settings()
#9 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Base_Object->ensure_settings()
#10 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/app.php(59): Elementor\Core\Base\Base_Object->get_settings()
#11 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/app.php(36): Elementor\Core\Base\App->get_components_config()
#12 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/app.php(153): Elementor\Core\Base\App->print_config()
#13 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(303): Elementor\Core\Common\App->register_scripts()
#14 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#15 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/plugin.php(470): WP_Hook->do_action()
#16 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-admin/includes/template.php(2046): do_action()
#17 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-admin/includes/plugin-install.php(598): iframe_header()
#18 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(303): install_plugin_information()
#19 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#20 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/plugin.php(470): WP_Hook->do_action()
#21 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-admin/plugin-install.php(70): do_action()
#22 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 09:14:37 [error X 1][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php::48] Uncaught ReflectionException: Method ElementorPro\Modules\ThemeBuilder\Documents\Section::get_site_editor_type() does not exist in /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php:48
Stack trace:
#0 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(48): ReflectionClass->getMethod()
#1 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(129): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document::get_site_editor_type_bc()
#2 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/categories/create.php(94): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document::get_create_url()
#3 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/categories/create.php(67): Elementor\Core\Common\Modules\Finder\Categories\Create->create_item_url_by_document_class()
#4 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/base-category.php(92): Elementor\Core\Common\Modules\Finder\Categories\Create->get_category_items()
#5 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Core\Common\Modules\Finder\Base_Category->get_init_settings()
#6 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Base_Object->ensure_settings()
#7 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/module.php(107): Elementor\Core\Base\Base_Object->get_settings()
#8 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Core\Common\Modules\Finder\Module->get_init_settings()
#9 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Base_Object->ensure_settings()
#10 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/app.php(59): Elementor\Core\Base\Base_Object->get_settings()
#11 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/app.php(36): Elementor\Core\Base\App->get_components_config()
#12 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/app.php(153): Elementor\Core\Base\App->print_config()
#13 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(303): Elementor\Core\Common\App->register_scripts()
#14 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#15 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/plugin.php(470): WP_Hook->do_action()
#16 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/script-loader.php(2083): do_action()
#17 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(303): wp_enqueue_scripts()
#18 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#19 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/plugin.php(470): WP_Hook->do_action()
#20 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/general-template.php(3015): do_action()
#21 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/themes/hello-elementor/header.php(21): wp_head()
#22 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/template.php(770): require_once('/var/www/www.xn...')
#23 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/template.php(716): load_template()
#24 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/general-template.php(48): locate_template()
#25 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/page-templates/templates/header-footer.php(9): get_header()
#26 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/template-loader.php(106): include('/var/www/www.xn...')
#27 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-blog-header.php(19): require_once('/var/www/www.xn...')
#28 /var/www/www.xn--byghndvrk-82ao.dk/www/index.php(17): require('/var/www/www.xn...')
#29 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 09:15:06 [notice X 676][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] Argumentet $control_id er forældet siden version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 09:15:15 [error X 3][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php::48] Uncaught ReflectionException: Method ElementorPro\Modules\ThemeBuilder\Documents\Section::get_site_editor_type() does not exist in /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php:48
Stack trace:
#0 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(48): ReflectionClass->getMethod()
#1 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(129): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document::get_site_editor_type_bc()
#2 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/categories/create.php(94): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document::get_create_url()
#3 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/categories/create.php(67): Elementor\Core\Common\Modules\Finder\Categories\Create->create_item_url_by_document_class()
#4 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/base-category.php(92): Elementor\Core\Common\Modules\Finder\Categories\Create->get_category_items()
#5 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Core\Common\Modules\Finder\Base_Category->get_init_settings()
#6 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Base_Object->ensure_settings()
#7 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/finder/module.php(107): Elementor\Core\Base\Base_Object->get_settings()
#8 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(142): Elementor\Core\Common\Modules\Finder\Module->get_init_settings()
#9 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/base-object.php(40): Elementor\Core\Base\Base_Object->ensure_settings()
#10 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/app.php(59): Elementor\Core\Base\Base_Object->get_settings()
#11 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/base/app.php(36): Elementor\Core\Base\App->get_components_config()
#12 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/app.php(153): Elementor\Core\Base\App->print_config()
#13 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(303): Elementor\Core\Common\App->register_scripts()
#14 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#15 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-includes/plugin.php(470): WP_Hook->do_action()
#16 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-admin/admin-header.php(102): do_action()
#17 /var/www/www.xn--byghndvrk-82ao.dk/www/wp-admin/plugins.php(603): require_once('/var/www/www.xn...')
#18 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 09:19:51 [notice X 8][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] Argumentet $control_id er forældet siden version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/jet-engine/framework/jet-elementor-extension/jet-elementor-extension.php(95): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2023-02-13 09:19:51 [notice X 8][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] Argumentet $control_id er forældet siden version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/jet-engine/framework/jet-elementor-extension/jet-elementor-extension.php(96): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2023-02-13 09:19:51 [notice X 180][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] Argumentet $control_id er forældet siden version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/dynamic-content-for-elementor/class/controls.php(64): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2023-02-14 14:46:07 [warning X 3][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-14 18:47:10 [warning X 4][/var/www/www.xn--byghndvrk-82ao.dk/www/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::76] Undefined array key "form_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 5 of 5JS: 2023-02-13 08:50:45 [error X 40][https://byghaandvaerk.dk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.2:3:782338] Cannot read properties of undefined (reading 'localized') 
JS: 2023-02-13 10:16:22 [error X 1][https://byghaandvaerk.dk/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.10.2:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-02-13 10:21:03 [error X 35][https://byghaandvaerk.dk/wp-content/plugins/dynamic-content-for-elementor/assets/js/visibility.js?ver=1.16.9:175:95] dce_model_cid is not defined 
JS: 2023-02-13 11:57:38 [error X 17][https://byghaandvaerk.dk/wp-admin/load-scripts.php?c=1&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-02-14 18:47:19 [error X 2][https://byghaandvaerk.dk/mentornetvaerk/:20:37] Invalid or unexpected token 



== Elementor - Compatibility Tag ==
	
	Dynamic.ooo - Dynamic Content for Elementor: Kompatibilitet ikke specificeret
	Elementor Pro: Kompatibilitet ikke specificeret

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by stefanullinger** [comment]
Having the same issue - file uploads not working anymore.

**Comment by hongsigner** [comment]
It worked just a few days ago, but it doesn't work now. I hope it's settled.

**Comment by trivia-studio** [comment]
I have exactly the same problem since I made the last update. A message simply displays "error" when submitting the form. Forms work just fine without adding file submissions.

**Comment by perks98** [comment]
Getting the same problem

**Comment by DanielShuster** [comment]
Same problem here.

**Comment by RKWO** [comment]
Same problem here.

**Comment by nicholaszein** [comment]
Hello everyone!

Thank you for your feedback! 🙏 

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11.1**! 🥳 

✔️ Please check it out and update your plugin to the new version!

Cheers 🥂
